playwright-e2e/tests/dsm/participant-page/participant-page-name.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, test } from '@playwright/test';
```

Comment:
Name of this test is `participant-page-name.spec.ts` inside `participant-page` dir. How about `participant-name-update.spec.ts` so it tells what test is testing.
Similar comment for `participant-page-tag.spec.ts`